src/components/SaturatedPoolAlert.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+import React from 'react';
+import type { Node } from 'react';
+import styled from 'styled-components';
+import Button from './common/Button';
+import type { DelegationProps } from '../containers/Home';
+
+const Wrapper = styled.div`
+  .section {
+    margin-bottom: 40px;
+  }
+  .title {
+    font-size: 18px;
+    line-height: 22px;
+    font-weight: bold;
+    color: #FF1351;
+  }
+  .description {
+    margin-top: 20px;
+    color: #2B2C32;
+    font-size: 14px;
+    line-height: 22px;
+  }
+  .upper-button {
+  }
+  .bottom-button {
+    margin-top: 20px;
+    background-color: #FF1351;
+    border: 2px solid #ff1351;
+    color: #fff;
+  }
+`;
+
+type Props = {|
+    delegation: {...DelegationProps, totalAda: ?number},
+    onSuccess: (id: string) => void,
+    close: () => void
+|}
+
+function SaturatedPoolAlert({ delegation, onSuccess, close }: Props): Node {
+  const text = delegation.isAlreadySaturated ?
+    `${delegation.stakepoolName} is already saturated. If you continue, you will get less than the expected rewards.`
+    :
+    `${delegation.stakepoolName} will be saturated after your delegation. If you continue, you will get less than the expected rewards.`
+
+  const poolShare = Number(delegation.stakepoolTotalStake) / 1000000;
```

Comment:
maybe mention 1000000 = 1ADA to avoid magic numbers? I noticed we use it in a few different places